products/compute/api.yaml

```diff
@@ -11452,32 +11527,742 @@ objects:
+                          prior to sending the response back to the client.
+                  - !ruby/object:Api::Type::Array
+                    name: 'matchRules'
+                    description: |
+                      The rules for determining a match.
+                    item_type: !ruby/object:Api::Type::NestedObject
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'fullPathMatch'
+                          description: |
+                            For satifying the matchRule condition, the path of the request must exactly
+                            match the value specified in fullPathMatch after removing any query parameters
+                            and anchor that may be part of the original URL. FullPathMatch must be between 1
+                            and 1024 characters. Only one of prefixMatch, fullPathMatch or regexMatch must
+                            be specified.
+                        - !ruby/object:Api::Type::Array
+                          name: 'headerMatches'
+                          description: |
+                            Specifies a list of header match criteria, all of which must match corresponding
+                            headers in the request.
+                          item_type: !ruby/object:Api::Type::NestedObject
+                            properties:
+                              - !ruby/object:Api::Type::String
+                                name: 'exactMatch'
+                                description: |
+                                  The value should exactly match contents of exactMatch. Only one of exactMatch,
+                                  prefixMatch, suffixMatch, regexMatch, presentMatch or rangeMatch must be set.
+                              - !ruby/object:Api::Type::String
+                                name: 'headerName'
+                                description: |
+                                  The name of the HTTP header to match. For matching against the HTTP request's
+                                  authority, use a headerMatch with the header name ":authority". For matching a
+                                  request's method, use the headerName ":method".
+                              - !ruby/object:Api::Type::Boolean
+                                name: 'invertMatch'
+                                default_value: false
+                                description: |
+                                  If set to false, the headerMatch is considered a match if the match criteria
+                                  above are met. If set to true, the headerMatch is considered a match if the
+                                  match criteria above are NOT met. Defaults to false.
+                              - !ruby/object:Api::Type::String
+                                name: 'prefixMatch'
```

Comment:
Same for these- can they be made exclusive?

Reply:
As above.